linear_search/Python/linear_search.py

```diff
@@ -0,0 +1,11 @@
+def linear_search(values, search_for):
+    search_at = 0
+    search_res = False
+    while search_at < len(values) and search_res is False:
+        if values[search_at] == search_for:
+            search_res = True
+        else:
```

Comment:
W391 blank line at end of file

*Origin: PycodestyleBear (W391), Section: `all.autopep8`.*